content/client-server-api/_index.md

```diff
@@ -363,6 +363,8 @@ specify parameter values. The flow for this method is as follows:
 
 {{% http-api spec="client-server" api="versions" %}}
 
+{{% http-api spec="client-server" api="support" %}}
```

Comment:
I'm not convinced this belongs under "Server discovery". I'd be inclined to put it in a new "Module", but I'm open to other suggestions

Reply:
Rationale behind the suggestion to put it here is you're *discovering* information about the server without necessarily needing to speak the rest of Matrix. Both authenticated and unauthenticated clients can use this, and it's "before" the homeserver APIs really start to do anything.

Arguably though, we should rename or introduce a section for `.well-known` stuff. I think this is best left to a future PR at this stage.